File: engine/engine-model/src/main/java/tools/packet/cashshop/operation/PutIntoCashInventory.java
package tools.packet.cashshop.operation;

import client.inventory.Item;
import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record PutIntoCashInventory(Item item, Integer accountId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_OPERATION;
   }
}

File: engine/model/src/main/scala/tools/packet/cashshop/operation/PutIntoCashInventory.scala
package tools.packet.cashshop.operation

import client.inventory.Item
import net.opcodes.SendOpcode
import tools.packet.PacketInput

class PutIntoCashInventory(private var _item: Item, private var _accountId: Int) extends PacketInput {
  def item: Item = _item

  def accountId: Int = _accountId

  override def opcode(): SendOpcode = SendOpcode.CASHSHOP_OPERATION
}